x/distribution/spec/05_hooks.md

```diff
@@ -5,26 +5,53 @@ order: 5
 # Hooks
 
 ## Create or modify delegation distribution
- 
+
  - triggered-by: `staking.MsgDelegate`, `staking.MsgBeginRedelegate`, `staking.MsgUndelegate`
 
-The pool of a new delegator bond will be 0 for the height at which the bond was
-added, or the withdrawal has taken place. This is achieved by setting
-`DelegationDistInfo.WithdrawalHeight` to the height of the triggering transaction. 
+### Before
+
+The delegation rewards are withdrawn to the withdraw address of the delegator.
+The rewards include the current period and exclude the starting period.
+The validator period is incremented.
+The validator period is incremented because the validator's power and share distribution might have changed.
+The reference count for the delegator's starting period is decremented.
```

Comment:
```suggestion
- The delegation rewards are withdrawn to the withdraw address of the delegator.
- The rewards include the current period and exclude the starting period.
- The validator period is incremented.
- The validator period is incremented because the validator's power and share distribution might have changed.
- The reference count for the delegator's starting period is decremented.
```

Reply:
using list items is helpful here

Reply:
line 14 doesn't seem like a hook (more of a description)

Reply:
It's not really a list, more of a paragraph.

Reply:
```suggestion
- The delegation rewards are withdrawn to the withdraw address of the delegator.
  The rewards include the current period and exclude the starting period.
- The validator period is incremented.
  The validator period is incremented because the validator's power and share distribution might have changed.
- The reference count for the delegator's starting period is decremented.
```